src/components/_reset.scss

```diff
@@ -7,16 +7,18 @@
     Remove all the styles of the "User-Agent-Stylesheet", except for the 'display' property
     - The "symbol *" part is to solve Firefox SVG sprite bug
  */
-*:where(:not(html, iframe, canvas, img, svg, video, audio):not(svg *, symbol
-      *)) {
-  all: unset;
-  display: revert;
-}
+// *:where(:not(html, iframe, canvas, img, svg, video, audio):not(svg *, symbol
+//       *)) {
+//   all: unset;
+//   display: revert;
+// }
 
 /* Preferred box-sizing value */
 *,
 *::before,
 *::after {
+  margin: 0;
```

Comment:
It's normal way that everyone do it and i saw in every project 
But you set all style globally to unset 
Because of that we couldn't see the style of markdown component.